.github/workflows/inspektor-gadget.yml

```diff
@@ -686,20 +687,22 @@ jobs:
     - uses: actions/checkout@v3
     - name: Create Release
       id: create_release
-      uses: actions/create-release@v1.0.0
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        tag_name: ${{ github.ref }}
-        release_name: Release ${{ github.ref }}
-        draft: false
-        prerelease: false
+      uses: softprops/action-gh-release@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        name: Release ${{ github.ref }}
     - name: Get all artifacts.
       uses: actions/download-artifact@v2
-    - name: Upload Gadget Release *-gadget-*-*.tar.gz
+    - name: Rename all artifacts to *-${{ env.GITHUB_REF_NAME }}.tar.gz
+      shell: bash
+      run: |
+        for i in *-gadget-*-*-tar-gz/*-gadget-*-*.tar.gz; do
+          mv $i $(dirname $i)/$(basename $i .tar.gz)-${GITHUB_REF_NAME}.tar.gz
+        done
```

Comment:
Could you please verify this step is working properly for the local-gadget binaries? Maybe just adding `build-local-gadget` as a dependency for the `release` job (just for testing).

Reply:
Thanks!